packages/config/config/devices/0x0258/nas-pd07z.json

```diff
@@ -10,63 +10,169 @@
 		}
 	],
 	"firmwareVersion": {
-		"min": "0.0",
+		"min": "2.6",
 		"max": "255.255"
 	},
 	"paramInformation": [
 		{
 			"#": "1",
-			"$import": "templates/shenzhen_neo_template.json#blink_on_motion"
+			"label": "Led indicator enable",
+			"valueSize": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "2",
-			"$import": "~/templates/master_template.json#base_enable_disable",
 			"label": "Motion Detection",
-			"defaultValue": 1
+			"valueSize": 1,
+			"unsigned": true,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "3",
-			"$import": "templates/shenzhen_neo_template.json#motion_prevent_retrigger"
+			"label": "Motion Alarm",
+			"description": "Report motion event alarm: 0 = every time motion event is detected,  1 = only once before motion event is cleared (default)",
+			"valueSize": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Every time",
+					"value": 0
+				},
+				{
+					"label": "Only once",
+					"value": 1
+				}
+			]
```

Comment:
And this is also just a more verbose variant of the imported template "Motion Sensor: Prevent Re-Trigger While On"